src/main/java/bio/overture/ego/model/enums/JavaFields.java

```diff
@@ -65,4 +65,5 @@
   public static final String REFRESH_ID = "refreshId";
   public static final String PROVIDERTYPE = "providerType";
   public static final String PROVIDERSUBJECTID = "providerSubjectId";
+  public static final String ERRORREDIRECTURI = "errorRedirectUri";
```

Comment:
Makes me want to refactor our naming convention here to use underscores cause I can't read this. `REFRESH_ID` uses underscores so maybe we can also use `ERROR_REDIRECT_URI`. Bonus points if you want to also update `PROVIDER_SUBJECT_ID` and others.

Reply:
@joneubank is right. Constants should use snake case upper caps.

Reply:
just in the JavaFields tho, right? (not sql fields...not looking to rename a bunch of columns atm :) )

Reply:
just the java variables. not the sql column names.